Assets/MixedRealityToolkit/Providers/BaseInputDeviceManager.cs

```diff
@@ -12,6 +13,13 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// </summary>
     public abstract class BaseInputDeviceManager : BaseDataProvider<IMixedRealityInputSystem>, IMixedRealityInputDeviceManager
     {
+        /// <summary>
+        /// Temporary control mechanism to enable/disable use of Pointer Cache in request/recycling of pointers by Input System
```

Comment:
Temporary? Do you mean to remove this during this PR, or after this PR?

Reply:
After this PR....I wish there was a #IF MRTK_DEV lol

I'll consider this more. At a min, I'll open a GitHub issue to track it's removal